src/pages/resources/mainnet/upgrades/v1.0.mdx

```diff
@@ -0,0 +1,74 @@
+# Mainnet upgrade: v1.0
+
+import { Callout } from '/src/components/callout'
+
+## About the v1.0 upgrade
+
```

Comment:
Mention what's being upgraded and to which specific version first (axelard v1.0.1 etc.). It's hidden at the bottom rn